scripts/stack/configure-stack.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
```

Comment:
While it's called `configure-stack` it only configures connectors, right? Does it make sense to rename it to reflect this?

Reply:
Yep - good point. Thanks!